spec/spec_helper.rb

```diff
@@ -39,6 +39,7 @@
 require 'capybara/rspec'
 require 'capybara/apparition'
 Capybara.javascript_driver = :apparition
+Capybara.default_driver = :apparition
```

Comment:
Je ne suis pas sûr que cela soit un comportement voulu. Cela reste lourd de lancer apparition pour tous les tests.

imo t'as du faire ça car la redirection sur https://auth-staging.api.gouv.fr doit merder non ? Dans ce cas là ajoute plutôt un `js: true` en metadata sur les tests de login.

Reply:
J'ai fait ça parce que le code suivant ne passait pas => `NoMethodError: undefined method `driven_by'`

Dans les specs de features : 
```ruby 
before { driven_by :apparition } 
```

Mes metadatas étaient `type: :feature` cependant.

Reply:
wtf is dat error ?? J'ai jamais vu ça.

je vais checker en local voir

Reply:
J'ai flip le test, sans le mock de OmniAuth il passe pas...